lighthouse-core/lib/log.js

```diff
@@ -46,10 +59,21 @@ class Log {
 
   static _logToStdErr(title, argsArray) {
     const args = [...argsArray];
-    if (!loggersByTitle[title]) {
-      loggersByTitle[title] = debug(title);
+    const log = Log.loggerfn(title);
+    log(...args);
+  }
+
+  static loggerfn(title) {
+    let log = loggersByTitle[title];
+    if (!log) {
+      log = debug(title);
+      loggersByTitle[title] = log;
+      // errors with red, warnings with yellow.
+      // eslint-disable-next-line no-nested-ternary
+      log.color = title.endsWith('error') ? colors.red :
```

Comment:
> nit extract error and warn string into named module private consts

disagree (maybe if we were exposing `Log.ERROR` but we have `log.error()` for that), but that shouldn't be a surprise by this point :)